#! /bin/sh /usr/share/dpatch/dpatch-run
## 001-nant-task-cs-InitializeElement.dpatch by Dave Beckett <dajobe@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Patch from CVS to handle C# compiler changing underneath us again.  Debian bug 422838

@DPATCH@
--- nant-0.85/src/NAnt.Core/Task.cs.orig	2007-06-24 09:54:35.000000000 -0700
+++ nant-0.85/src/NAnt.Core/Task.cs	2007-06-24 09:58:04.000000000 -0700
@@ -400,7 +400,6 @@
         #region Protected Instance Methods
 
         /// <summary><note>Deprecated (to be deleted).</note></summary>
-        [Obsolete("Deprecated- Use InitializeTask instead")]
         protected override void InitializeElement(XmlNode elementNode) {
             // Just defer for now so that everything just works
             InitializeTask(elementNode);
@@ -415,4 +414,4 @@
 
         #endregion Protected Instance Methods
     }
-}
\ No newline at end of file
+}
